alacritty/src/display/mod.rs

```diff
@@ -622,6 +553,37 @@ impl Display {
         })
     }
 
+    #[inline]
+    pub fn gl_context(&self) -> &PossiblyCurrentContext {
+        self.context.get()
+    }
+
+    pub fn make_not_current(&mut self) {
+        if self.context.get().is_current() {
+            self.context.replace_with(|context| {
+                context.make_not_current().expect("failed to disable context").treat_as_current()
```

Comment:
"make_not_current" -> "treat_as_current" sounds **very** wrong.

Reply:
Maybe the name is wrong? We're doing that already, nothing new here. Given that your context is possibly current and you basically tell it to stay like that and not go into `NotCurrent`.

Reply:
Maybe the function should be renamed to `treat_as_possibly_current`? That way it's clear that we don't assume it's current but we just treat it as a context that might be?